content/docs/command-reference/add.md

```diff
@@ -56,11 +56,6 @@ each one:
 Summarizing, the result is that the target data is replaced by small `.dvc`
 files that can be easily tracked with Git.
 
-> Note that `.dvc` files can be considered _orphan stages_, because they have no
-> <abbr>dependencies</abbr>, only outputs. These are treated as _always changed_
-> by `dvc status` and `dvc repro`, which always executes them. See `dvc.yaml` to
-> learn more about stages.
-
 To avoid adding files inside a directory accidentally, you can add the
```

Comment:
this should move or copy into the Tracking directories? WDYT

Reply:
Good idea. I guess this note should be more general about .dvcignore (with link to that guide) and move this text to the Directory section ⌛

Reply:
Done in b3eb4631 but let me double check whether .dvcignore actually prevents explicit adds... ⌛

Reply:
Yep. It's not allowed (`ERROR: output 'data' does not exist` message).

> See also iterative/dvc/issues/3538